src/main/java/com/microsoft/sqlserver/jdbc/SQLServerPreparedStatement.java

```diff
@@ -69,6 +69,7 @@
     /** True if this execute has been called for this statement at least once */
     private boolean isExecutedAtLeastOnce = false;
 
+    private boolean needsPrepare = false;
```

Comment:
Why do we need `needsPrepare` as a instance field? Should always be local variable.

Reply:
Fixed.

Reply:
Great!